go-controller/pkg/cni/ovs_unit_test.go

```diff
@@ -62,39 +55,27 @@ func TestOvsExec(t *testing.T) {
 		{
 			desc:                "Test codepath when runner is non nil and returns error",
 			expectedErr:         fmt.Errorf("failed to run ovs-vsctl"),
-			onRetArgsKexecIface: &ovntest.TestifyMockHelper{"Command", []string{"string", "string"}, []interface{}{mockCmd}},
-			onRetArgsCmdList:    &ovntest.TestifyMockHelper{"CombinedOutput", []string{}, []interface{}{nil, fmt.Errorf("failed to run 'ovs-vsctl")}},
+			onRetArgsKexecIface: &ovntest.TestifyMockHelper{OnCallMethodName: "Command", OnCallMethodArgType: []string{"string", "string"}, RetArgList: []interface{}{mockCmd}},
+			onRetArgsCmdList:    &ovntest.TestifyMockHelper{OnCallMethodName: "CombinedOutput", OnCallMethodArgType: []string{}, RetArgList: []interface{}{nil, fmt.Errorf("failed to run 'ovs-vsctl")}},
 			runnerInstance:      mockKexecIface,
 		},
 		{
 			desc:                "Test codepath when runner is not nil and does not return an error",
 			expectedErr:         nil,
-			onRetArgsKexecIface: &ovntest.TestifyMockHelper{"Command", []string{"string", "string"}, []interface{}{mockCmd}},
-			onRetArgsCmdList:    &ovntest.TestifyMockHelper{"CombinedOutput", []string{}, []interface{}{nil, nil}},
+			onRetArgsKexecIface: &ovntest.TestifyMockHelper{OnCallMethodName: "Command", OnCallMethodArgType: []string{"string", "string"}, RetArgList: []interface{}{mockCmd}},
+			onRetArgsCmdList:    &ovntest.TestifyMockHelper{OnCallMethodName: "CombinedOutput", OnCallMethodArgType: []string{}, RetArgList: []interface{}{nil, nil}},
 			runnerInstance:      mockKexecIface,
 		},
 	}
 	for i, tc := range tests {
 		t.Run(fmt.Sprintf("%d:%s", i, tc.desc), func(t *testing.T) {
+			//ovntest.PreProcessMockFn(&mockKexecIface.Mock, *tc.onRetArgsKexecIface)
```

Comment:
nit: remove these comments

Reply:
@JacobTanenbaum thanks for the catch, shall remove them